LoopBackTests/LBPersistedModelDataTypesTests.m

```diff
@@ -0,0 +1,232 @@
+//
+//  LBPersistedModelDataTypesTests.m
+//  LoopBack
+//
+//  Created by hideya kawahara on 1/6/16.
+//  Copyright © 2016 StrongLoop. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+
+#import "LBPersistedModel.h"
+#import "LBRESTAdapter.h"
+#import "SLRemotingTestsUtils.h"
+
+
+@interface Widget2 : LBPersistedModel
```

Comment:
The reason why I needed to use the name `Widget2` is that the class name `Widget` has been used in a different unit test `LBPersistedModelSubclassingTests.m` and I needed to avoid conflict in names (note that `LBPersistedModelDataTypesTests.m` is separated from `LBPersistedModelSubclassingTests.m` for readability).  The difference between them is that `Widget2` defines more properties than `Widget` for testing property handling.

So, how about using `WidgetEx` instead of `Widget2`? (where `Ex` stands for extra properties)